### ISSUE TITLE
Ins and Del elements not working inside buttons

### ISSUE THREAD

**Issue by jawsuser** [FreedomScientific/VFO-standards-support#888]
Jaws normally reads out the ins and del elements. Through some recent testing I discovered that if the ins and del elements are part of a button they are not read out by jaws. This happens on both edge and chrome. Tested it with NVDA and the ins and del elements work as expected, so it looks like this is a bug with jaws. This occurs using Jaws 2024.2403.3

**Comment by JAWS-test** [comment]
The cause could also be the ARIA specification: https://w3c.github.io/aria/#button, where it is defined that everything within a button has role=presentation

**Comment by JeremiahRogersVispero** [comment]
While I was able to recreate the issue, and I understand @jawsuser's position, the "children presentational" declaration within the ARIA specification means elements such as <ins> and <del> should be ignored by AT.
